cli/client.go

```diff
@@ -622,6 +626,13 @@ func RobotsLogsAction(c *cli.Context, args robotsLogsArgs) error {
 		return err
 	}
 
+	// Check if both start time and count are provided
+	if args.Start != "" && args.Count > 0 {
+		return fmt.Errorf("unsupported functionality: specifying both a start time and a count is not supported. " +
+			"Alternative functionality is to provide start time and end time",
```

Comment:
can we make this a little more descriptive? and explain why its not supported (i.e. the weird UX you noted)